src/Winium.Cruciatus/Application.cs

```diff
@@ -109,9 +122,37 @@ public void Start(string arguments)
                                Arguments = arguments
                            };
 
-            this.process = Process.Start(info);
+            this.process = Process.Start(info);           
         }
 
+        /// <summary>
+        /// Update process property by process name
+        /// </summary>
```

Comment:
from @zelonght This is like hardcoded, any in future we may have other similar apps like this to append?
		Any general solution for this? Should do simple way, int case many process return without doing anything